media-sound/wavpack/wavpack-5.0.0.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="Open audio compression format"
```

Comment:
The great feature of WavPack is its hybrid format, so maybe this could be a slightly summary:
```
SUMMARY="Hybrid lossless audio compression tools & format"
```
What do you think?
Hmm, do we drop the " & format" ?